README.md

```diff
@@ -78,4 +78,16 @@ git push origin v0.0.1
 
 	`$ rosa-support delete vpc --vpc-id <vpc id> --region <region>`
 
+* Prepare bastion
+	* *--region* is required where created the vpc-id
+	* *--availability-zone* is required on which zone to launch the bastion instance
+    * *--vpc-id* is required which should be VPC id used to launch cluster
```

Comment:
Is this expected to have less whitespaces from other lines?

Reply:
Updated.